t/apicast-policy-upstream-connection.t

```diff
@@ -59,3 +61,88 @@ GET /
 --- error_log
 upstream timed out
 --- error_code:
+
+=== TEST 2: Set timeouts using HTTPS proxy for backend
```

Comment:
Can we also add a test case when using `upstream_connection` with camel proxy and with proxy env variables?